platforms/access-bridge/test/unit/errors.ts

```diff
@@ -0,0 +1,79 @@
+import assert from 'assert';
+import { describe, test } from 'node:test';
+
+import {
+  BadRequestError,
+  ErrorCode,
+  ForbiddenError,
+  getErrorCodeDescription,
+  InternalError,
+  MethodNotAllowedError,
+  NotFoundError,
+  ParameterInvalidError,
+  ParameterMissingError,
+  UnauthorizedError,
+} from '../../src/errors.js';
+import { RequestMethod } from '../../src/http.js';
+
+describe('Error Classes', () => {
```

Comment:
I'm not sure how valuable these tests are. I would prefer to test the error codes through a response path that throws the expected type of error. Otherwise, it's a bunch of boilerplate to test a bunch of boilerplate.